tools/downloader/common.py

```diff
@@ -51,6 +51,7 @@
 }
 KNOWN_TASK_TYPES = {
     'action_recognition',
+    'audio_classification',
```

Comment:
Please update `README.txt` accordingly.

Reply:
Done

Reply:
@eizamaliev it seems task type should be changed to `sound_classification` what is generally used name of such kind of tasks according to[ comment](https://github.com/opencv/open_model_zoo/pull/1258#pullrequestreview-448981525)

Reply:
@sathishkuttan Can you recommend final naming of task? From our (DLStreamer) perspective we will describe it as audio_event_detection - but don't see a concern with it being listed here as sound / audio classification.
@vladimir-dudnik From a brief literature survey there doesn't seem to be as much standardized naming as for CV - but it was only a brief look

Reply:
https://arxiv.org/pdf/1811.06669v1.pdf 

Uses Audio Classification

Reply:
Let's go with "sound classification" which is what we have used in the internal efforts (JIRA, internal repo, model directory in download.01, model description etc.)

Reply:
@sathishkuttan  - will you update the files accordingly? Currently looks like audio classification

Reply:
@nnshal1 I authored only the documentation file (`aclnet.md`) where I used sound classification consistent with my comment. Other files were added by @eizamaliev I think.
@eizamaliev can you please change the naming accordingly?

Reply:
@sathishkuttan @vladimir-dudnik @IRDonch 

Changes made - please resolve thread